sickbeard/tv.py

```diff
@@ -166,9 +166,7 @@ def network_logo_name(self):
 
     @property
     def is_recently_deleted(self):
```

Comment:
Nah, but it makes for readable code. As now your 'forced' to use the TVobject, as opposed to do the check on a random indexerid. Also in the future if you want to do a more advanced check, or make changes to the way you check if a show has been deleted, you only need to change here.